### PR TITLE
Adding days since match to match details page

### PR DESCRIPTION
**Pull request by ccaprio97**
## What’s changing?

image

## Why?

NAC-167

## This PR has:

- [X] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [ ] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [x] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [ ] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

### CODE REVIEW

File: query-tool/tests/Piipan.QueryTool.Tests/Components/MatchDetail/StateDispositionInformationTests.razor [hunk 1]
```diff
@@ -36,12 +36,78 @@
...
         );
     }
 
     #region Tests
 
+    /// <summary>
+    /// Verify the default markup of the state disposition information
+    /// </summary>
+    [Fact]
+    public void State_Contact_Info_Should_Exist_And_Have_Correct_Markup_With_Match_Date()
```
Comment (bgfoshay) [inline query-tool/tests/Piipan.QueryTool.Tests/Components/MatchDetail/StateDispositionInformationTests.razor:61]: Question/Thought: We probably should consider a test for the case when Match Date is empty/null/invalid

Comment (ccaprio97) [reply query-tool/tests/Piipan.QueryTool.Tests/Components/MatchDetail/StateDispositionInformationTests.razor:61]: the rest of them do that. thats why i had to make this one

Comment (bgfoshay) [reply query-tool/tests/Piipan.QueryTool.Tests/Components/MatchDetail/StateDispositionInformationTests.razor:62]: 👍

**Review by bgfoshay — APPROVED** [review]
LGTM

**Pull request merged**